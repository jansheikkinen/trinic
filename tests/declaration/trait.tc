// Trait

trait TestTrait
  function testFunction(void) -> void
  function testFunctionB(void) -> void
end
