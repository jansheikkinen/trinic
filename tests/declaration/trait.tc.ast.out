
### AST GENERATION ###
Declaration TRAIT TestTrait
  Declaration FUNCTION  testFunction
  Declaration FUNCTION  testFunctionB
